src/engraving/rw/read114/read114.cpp

```diff
@@ -2729,9 +2729,13 @@ Err Read114::readScore(Score* score, XmlReader& e, ReadInOutData* out)
     }
 
     ReadContext ctx(score);
-    if (out && out->overriddenSpatium.has_value()) {
-        ctx.setSpatium(out->overriddenSpatium.value());
-        ctx.setOverrideSpatium(true);
+    if (out) {
+        if (out->overriddenSpatium.has_value()) {
+            ctx.setSpatium(out->overriddenSpatium.value());
+            ctx.setOverrideSpatium(true);
+        }
+
+        ctx.setStylesToSkip(std::move(out->stylesToSkip));
```

Comment:
I think creating a copy here is better than using std::move. It's a bit odd that stylesToSkip will be invalid after calling readScore